src/components/Dialogs/AdvancedDelegateDialog/AdvancedDelegateDialog.tsx

```diff
@@ -11,7 +10,7 @@ import {
 } from "react";
 import { useAccount } from "wagmi";
 import { Delegation } from "@/app/api/common/delegations/delegation";
-import { ChevronsRight, DivideIcon, InfoIcon, Repeat2 } from "lucide-react";
+import { DivideIcon, InfoIcon, Repeat2 } from "lucide-react";
```

Comment:
whats your preference between lucide-react and heroicons? I feel like it's overkill that we have both. I don't know if its harming perf or anything, probably not. But I notice they have many of the same icons

Reply:
(I typically use heroicons but I don't have much opinion, just been doing that for a while since it's often used in tailwind examples)